docs/source/concepts/ledger-model/ledger-integrity.rst

```diff
@@ -17,19 +17,22 @@ Valid Ledgers
 At the core is the concept of a *valid ledger*; changes
 are permissible if adding the corresponding commit to the
 ledger results in a valid ledger. **Valid ledgers** are
-those that fulfill three conditions:
+those that fulfill four conditions:
 
 :ref:`da-model-consistency`
```

Comment:
Should this be updated (or the heading it refers to) to "Ledger Consistency", so as not to get confused with "Key Consistency"?

Reply:
Folded the two consistency conditions into one section.